r-package/grf/tests/testthat/test_forest_summaries.R

```diff
@@ -175,4 +178,21 @@ test_that("best linear projection is reasonable", {
 
   expect_lt(mean(abs(std.errs)), 2.5)
   expect_gt(mean(abs(std.errs)), 0.25)
+
+  blp.shifted <- best_linear_projection(forest.shifted.W, X[,1:2])
```

Comment:
I might rename blp.s to blp.subset earlier, since you now also have shifted- easier to read.